src/renderer/components/settings-credits.tsx

```diff
@@ -19,9 +18,9 @@ interface Contributor {
   api: string;
   login: string;
   avatar: string;
-  name: string;
-  bio: string;
-  location: string;
+  name: string | null;
+  bio: string | null;
+  location: string | null;
```

Comment:
Are these required changes to make things work, or just updating this to be more accurate? If the latter, let's not include it in this PR, can be done in a follow-up. Since this is already a large PR, I'd like to keep things focused.

Reply:
I had to add these as within test `mockContributorsBroken` is defined holding these values as `null` as well (It gives an type error when calling the method - `getContributors`)

Reply:
But now with using `wrapper.setState({ contributors })`, This isn't required, so I will be reverting this :)